spynnaker/pyNN/extra_algorithms/splitter_components/abstract_spynnaker_splitter_delay.py

```diff
@@ -26,17 +26,14 @@ class AbstractSpynnakerSplitterDelay(object, metaclass=AbstractBase):
 
     __slots__ = []
 
-    # max delays supported by a slice split machine vertex
-    MAX_SUPPORTED_DELAY_TICS = 16
-
```

Comment:
This simplifies the issues I've been having on other branches somewhat so I'm happy to see it go in.  I wonder what the effect of this is in general on networks which previously used delay extensions, and whether it means that we might need some further integration testing such that delay extensions are still tested properly?

Reply:
Yes worth a check in general

Reply:
Looks like a few tests are failing because of some assumptions about delay extensions so it's worth looking into them.

Reply:
These are passing now.  The issue was more about clearing the delay status in the splitter.